doc/nrf/known_issues.rst

```diff
@@ -341,6 +342,14 @@ Missing :file:`CMakeLists.txt`
 Thread
 ======
 
+.. rst-class:: v1-9-0
+
+KRKNWK-13059: Wrong MAC frame counter is reported sometimes
+  The reporting of the wrong MAC frame counter causes the neighbor to drop subsequent frames from the device due to security checks.
+  This issue only affects to Thread 1.2 builds.
+
+  **Workaround:** To fix the issue, cherry-pick the commit with the hash ``1ab6be252335ceec5a966b36fbc79883ebd1c4d1``.
```

Comment:
Need to specify that this is a `sdk-zephyr` commit?

Reply:
I thougth you gave me the commit hash to `sdk-nrf`...

Reply:
Updated as per conversation on Teams.